src/tests/owners.rs

```diff
@@ -203,36 +203,26 @@ fn owners_can_remove_self() {
 */
 #[test]
 fn check_ownership_two_crates() {
-    let (_b, app, middle) = app();
+    let (app, anon, user) = TestApp::with_user();
 
-    let (krate_owned_by_team, team) = {
-        let conn = app.diesel_database.get().unwrap();
-        let u = new_user("user_foo").create_or_update(&conn).unwrap();
-        let t = new_team("team_foo").create_or_update(&conn).unwrap();
-        let krate = CrateBuilder::new("foo", u.id).expect_build(&conn);
-        add_team_to_crate(&t, &krate, &u, &conn).unwrap();
+    let (krate_owned_by_team, team) = app.db(|conn| {
+        let t = new_team("team_foo").create_or_update(conn).unwrap();
+        let krate = CrateBuilder::new("foo", user.as_model().id).expect_build(conn);
+        add_team_to_crate(&t, &krate, user.as_model(), conn).unwrap();
         (krate, t)
-    };
-
-    let (krate_not_owned_by_team, user) = {
-        let conn = app.diesel_database.get().unwrap();
-        let u = new_user("user_bar").create_or_update(&conn).unwrap();
-        (::CrateBuilder::new("bar", u.id).expect_build(&conn), u)
-    };
-
-    let mut req = req(Method::Get, "/api/v1/crates");
+    });
 
-    let query = format!("user_id={}", user.id);
-    let mut response = ok_resp!(middle.call(req.with_query(&query)));
-    let json: CrateList = ::json(&mut response);
+    let user2 = app.db_new_user("user_bar");
```

Comment:
hoooo boy this test's setup code was confusing before, with 2 `u`s and a `user`. much better now 👍